source/dotnet/Test/AdaptiveCards.Test/AdaptiveCardApiTests.cs

```diff
@@ -1245,5 +1247,161 @@ public void TestParsingRichTextBlockWithInvalidInlineType()
 
             StringAssert.Contains(ex.Message, "Property 'type' must be 'TextRun'");
         }
+
+        void RenderCardTask(string payload)
+        {
+            try
+            {
+                AdaptiveCardParseResult parseResult = AdaptiveCard.FromJson(payload);
+            } catch (Exception ex)
+            {
+                lock (this)
+                {
+                    exceptions.Add(ex);
+                }
+            }
+        }
+
+        [TestMethod]
+        public void TestConcurrentParsing()
+        {
+            // card with invalid inline type
+            var adaptiveCard =
+@"{
+	""$schema"": ""http://adaptivecards.io/schemas/adaptive-card.json"",
+	""type"": ""AdaptiveCard"",
+	""version"": ""1.0"",
+	""body"": [
+		{
+			""type"": ""TextBlock"",
```

Comment:
>			 [](start = 0, length = 3)

looks like we've got tabs here

Reply:
thanks

---
In reply to: [340284336](https://github.com/microsoft/AdaptiveCards/pull/3546#discussion_r340284336) [](ancestors = 340284336)